packages/earn-controller/src/index.ts

```diff
@@ -15,3 +17,20 @@ export {
   getDefaultEarnControllerState,
   EarnController,
 } from './EarnController';
+
+export {
+  selectLendingMarkets,
+  selectLendingPositions,
+  selectLendingMarketsWithPosition,
+  selectLendingPositionsByProtocol,
+  selectLendingMarketByProtocolAndTokenAddress,
+  selectLendingMarketForProtocolAndTokenAddress,
+  selectLendingPositionsByChainId,
+  selectLendingMarketsByChainId,
+  selectLendingMarketsByProtocolAndId,
+  selectLendingMarketForProtocolAndId,
+  selectLendingPositionsWithMarket,
+  selectLendingMarketsForChainId,
+} from './selectors';
+
+export * from '@metamask/stake-sdk';
```

Comment:
Hmm... what is your motivation for re-exporting everything from `@metamask/stake-sdk` from this package? It seems that blanket-exporting a bunch of stuff could bloat the size of this package without realizing it. Could consuming projects just use what they need from `@metamask/stake-sdk` directly?

Reply:
I have updated to include a few exports for now.